pages/linux/slop.md

```diff
@@ -0,0 +1,24 @@
+# slop
+
+> Get a selection of the screen.
+> More information: <https://github.com/naelstrof/slop>.
+
+- Wait for the user to make a selection and output its geometry to standard output:
+
+`slop`
+
+- Double click, rather than click and drag, to draw a selection:
+
+`slop -D`
+
+- Highlight, rather than outline, the selection:
+
+`slop -l`
+
+- Specify output format:
```

Comment:
Perhaps this will read better?
```suggestion
- Specify the output format:
```

Also, is there a limited number of supported output formats? We can update example itself to show `{{foo|bar|baz}}` instead of `{{format}}`, for example.

Reply:
Formats are given as a format string e.g. `"%xx%y"`. Maybe rename `{{format}}` to `{{format_string}}`?

Reply:
Resolved in 5fc750e4f8f1e0b717a30bdf979cc3f3a19436ad

Reply:
Thanks for that commit, @MarkusG - but this comment is about a grammatical error.

Reply:
Ah, oops! Didn't catch that.

Reply:
Resolved in 84aec1cb576da76a27f2d9947708b7ceb95416d2